website/community/committers/release.md

```diff
@@ -596,3 +596,20 @@ fully endorsed by the ASF.
 ```
 
 Example: <https://lists.apache.org/thread/oy77n55brvk72tnlb2bjzfs9nz3cfd0s>
+
+## Post release
+
+After the official release out, you may perform a few post actions.
+
+### Remove the old releases
+
+Remove the old releases if any. You only need the latest release there, and older releases are available through the Apache archive.
```

Comment:
How many versions do we want to leave on the download page?
Only the last one may not be a good idea.🤔

cc @Xuanwo

Reply:
We don't do backport. So let's maintain only the latest one.